docroot/sites/classrooms.uiowa.edu/modules/classrooms_core/classrooms_core.module

```diff
@@ -31,6 +55,199 @@ function classrooms_core_form_alter(&$form, FormStateInterface $form_state, $for
       $form['field_room_scheduling_regions']['#disabled'] = TRUE;
       break;
 
+    case 'taxonomy_term_technology_features_form':
+
+      // Disable font awesome extra settings.
+      /** @var Drupal\uiowa_core\Access\UiowaCoreAccess $check */
+      $check = \Drupal::service('uiowa_core.access_checker');
+
+      /** @var Drupal\Core\Access\AccessResultInterface $access */
+      $access = $check->access(\Drupal::currentUser()->getAccount());
+
+      if ($access->isForbidden()) {
+        $form['field_technology_features_icon']['widget'][0]['settings']['size']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['fixed-width']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['border']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['invert']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['animation']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['additional_classes']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['duotone']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['masking']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['power_transforms']['#disabled'] = TRUE;
+        $form['field_technology_features_icon']['widget'][0]['settings']['pull']['#disabled'] = TRUE;
```

Comment:
Is this effectively to hide the entire `$form['field_technology_features_icon']['widget'][0]['settings']` part of the form?

Reply:
All of those are listed separately because disabling the `style` option seemed to break the functionality for webmasters.

Reply:
Does `$form['field_technology_features_icon']['widget'][0]['settings']['#disabled'] = TRUE;` not work?

Reply:
It does, but we seem to need `$form['field_technology_features_icon']['widget'][0]['settings']['style']` visible in order for the icons to render for webmasters.

Reply:
I'm also thinking you could do something like:
```php
use Drupal\Core\Render\Element;

foreach (Element::children($form['field_technology_features_icon']['widget'][0]['settings']) as $index) {
  $form['field_technology_features_icon']['widget'][0]['settings'][$index]['#disabled'] = TRUE;
}
```

Reply:
Updated to account for @bspeare's last comment:

```php
use Drupal\Core\Render\Element;

foreach (Element::children($form['field_technology_features_icon']['widget'][0]['settings']) as $index) {
  if ($index === 'style') {
    continue;
  }
  $form['field_technology_features_icon']['widget'][0]['settings'][$index]['#disabled'] = TRUE;
}
```

Reply:
Thanks for the suggestion! The code has been updated.